resources/zib/WoundImage.xml

```diff
@@ -0,0 +1,116 @@
+<?xml version="1.0" encoding="utf-8"?>
+<StructureDefinition xmlns="http://hl7.org/fhir">
+  <id value="WoundImage" />
+  <url value="http://nictiz.nl/fhir/StructureDefinition/WoundImage" />
+  <name value="WoundImage" />
+  <title value="Wound Image" />
+  <status value="draft" />
+  <publisher value="Nictiz" />
+  <contact>
+    <name value="Nictiz" />
+    <telecom>
+      <system value="url" />
+      <value value="https://www.nictiz.nl" />
+      <use value="work" />
+    </telecom>
+  </contact>
+  <description value="This WoundImage profile encapsulates the WoundImage idea as it applies to three different zibs:&#xD;&#xA;&#xD;&#xA;* [Burnwound v3.4(2020EN)](https://zibs.nl/wiki/Burnwound-v3.4(2020EN))&#xD;&#xA;* [Wound v3.3(2020EN)](https://zibs.nl/wiki/Wound-v3.3(2020EN))&#xD;&#xA;* [PressureUlcer v3.3(2020EN)](https://zibs.nl/wiki/PressureUlcer-v3.4(2020EN))" />
+  <purpose value="A separate documentReference profile for exchanging wound images." />
+  <copyright value="Copyright and related rights waived via CC0, https://creativecommons.org/publicdomain/zero/1.0/. This does not apply to information from third parties, for example a medical terminology system. The implementer alone is responsible for identifying and obtaining any necessary licenses or authorizations to utilize third party IP in connection with the specification or otherwise." />
+  <fhirVersion value="4.0.1" />
+  <mapping>
+    <identity value="zib-burnwound-v3.4-2020EN" />
+    <uri value="https://zibs.nl/wiki/Burnwound-v3.4(2020EN)" />
+    <name value="zib Burnwound-v3.4(2020EN)" />
+  </mapping>
+  <mapping>
+    <identity value="zib-pressureulcer-v3.4-2020EN" />
+    <uri value="https://zibs.nl/wiki/PressureUlcer-v3.4(2020EN)" />
+    <name value="zib PressureUlcer-v3.4(2020EN)" />
+  </mapping>
+  <mapping>
+    <identity value="zib-wound-v3.3-2020EN" />
+    <uri value="https://zibs.nl/wiki/Wound-v3.3(2020EN)" />
+    <name value="zib Wound-v3.3(2020EN)" />
+  </mapping>
+  <kind value="resource" />
+  <abstract value="true" />
+  <type value="DocumentReference" />
+  <baseDefinition value="http://hl7.org/fhir/StructureDefinition/DocumentReference" />
+  <derivation value="constraint" />
+  <differential>
+    <element id="DocumentReference.content">
```

Comment:
Is it really necessary to slice `.content` here? You could just add all three mappings on the same `.content.attachment.url`, and combine the `.definition` (see the changes I've made to the `.description`).

Reply:
Slices deleted